modules/ROOT/pages/message-type-receive-from-partners.adoc

```diff
@@ -1,10 +1,16 @@
 = Receive from Partners Message Type Settings
 
-The Receive from Partners message type defines the way in which Anypoint Partner Manager transforms the inbound transactions sent from your partner to your organization. These transactions typically follow JSON, EDIFACT, X12, or XML standards.
+:message-type: Receive from Partners
+
+The Receive from Partners message type defines the way in which Anypoint Partner Manager transforms the inbound transactions sent from your partner to your organization. You can define the CSV, EDIFACT, JSON, X12, or XML format for the Receive from Partners message type.
```

Comment:
If you define an attribute, wouldn't you use it wherever you mention that message type? (I need to learn more about attributes, obvs.)

Reply:
I am not understanding your question.